composer.json

```diff
@@ -12,6 +12,7 @@
     }
   ],
   "require": {
+    "php": "^5.5 || ^7",
```

Comment:
IMHO that is quite unusal 
Why just not to specify minimum version?
      "php": ">=5.5"